From 8f4654a5a7b78809dd3da9b2a3434426f8c7e53d Mon Sep 17 00:00:00 2001
From: Colin Xu <colin.xu@intel.com>
Date: Wed, 2 Sep 2020 13:38:40 +0800
Subject: [PATCH] VM_MANAGER: Update setup_host.sh to install acpica-tools

The script requires iasl installed to compile ACPI SSDT.
Install acpica-tools package to get the tool.

Change-Id: Iad5f9a43c08c135a3886c89fb2ce31f763ba6b7a
Tracked-On: OAM-92664
Signed-off-by: Colin Xu <colin.xu@intel.com>
---
 scripts/setup_host.sh | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/scripts/setup_host.sh b/scripts/setup_host.sh
index ea6172208738..9c9d06c4203a 100755
--- a/scripts/setup_host.sh
+++ b/scripts/setup_host.sh
@@ -28,7 +28,7 @@ function ubu_changes_require(){
     if [ x$res = xn ]; then
         return 1
     fi
-    sudo apt install -y wget mtools ovmf dmidecode python3-usb python3-pyudev pulseaudio jq
+    sudo apt install -y wget mtools ovmf dmidecode python3-usb python3-pyudev pulseaudio jq acpica-tools
 }

 function ubu_install_qemu_gvt(){
--
2.28.0

